distribution/docs/src/main/resources/_contents/_draft-registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,369 @@
+
+.[[service_bindings]]Services
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Service Name
+|This service name
+|string
+|
+
+|Service Description
+|Short description for this service
+|string
+|
+
+|Service Version
+|This service version
+|string
+|
+
+|Service Type
+|Identifies the type of service this is by a URN.
+|string
+|
+
+4+^|Bindings (Click *Add* to add a service binding)
+
+|Binding Name
+|This binding name
+|String
+|yes
+
+|Binding Description
+|Short description for this binding
+|String
+|
+
+|Binding Version
+|This binding version
+|
+|
+
+|Access URL
+|The url used to access this binding
+|
+|
+
+|Service Binding Type
+|The binding type for the service
+|
+|
+
+|URL Property Key
+|Property that the accessURI value should be put into for source creation
+|
+|
+
+4+|
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+.Organizations Tab (click *Add* to add an organization)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Organization Name
+|This organization's name
+|string
+|yes
+
+|Address
+|This organization's primary address
+|Expand to enter address information
+|yes
+
+|TelephoneNumber
+|Primary contact number for this organization
+|
+|no
+
+|Email
+|Primary contact email for this organization
+|
+|no
+
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+.Contacts (click *Add* button to add contact info)
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Contact Title
+|Contact Title
+|String
+|yes
+
+|Contact First Name
+|Contact First Name
+|String
+|yes
+
+|Contact Last Name
+|Contact Last Name
+|String
+|yes
+
+|Address
+|Address for listed contact
+|String
+|minimum one
+
+|Phone number
+|Contact phone number
+|
+|minimum one
+
+|Email
+|Contact email
+|String
+|minimum one
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+.Collections (Click *Add* to add Content Collection(s))
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Content Name
+|Name for this metadata content
+|string
+|yes
+
+|Content Description
+|Short description for this metadata content
+|string
+|no
+
+|Content Object Type
+|The kind of content object this will be. Default value should be used in most cases.
+|string
+|yes
+
+|Custom Fields
+a|click *Add* button to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button to add associations
+|Configurable
+|no
+
+|===
+
+===== Adding a Service Binding to a Node
+
+Advertise the methods other nodes use to connect to the local ${branding} instance.
+
+* Navigate to ${admin-console}.
```

Comment:
I prefer numbered steps. I've been trying to change them. There are some cases where it can't handle numbered lists with interruptions, but that's not the case here. Fixed.